AsyncDisplayKit/ASTableView.mm

```diff
@@ -303,7 +303,7 @@ - (void)reloadDataWithCompletion:(void (^)())completion
   ASPerformBlockOnMainThread(^{
     [super reloadData];
```

Comment:
Not sure if possible, but I really want to eliminate this call. We will do this afterward anyway.